solutions/USACO jan19-mountain-view.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: usaco-896
+source: USACO Silver 2019 January
+title: Mountain View
+author: Jesse Choe
+---
+
+[Official Analysis](http://usaco.org/current/data/sol_prob3_silver_dec20.html)
+
+**Time Complexity:** $\mathcal{O}(n\log n)$
+
+<LanguageSection>
+<CPPSection>
+
+## Sorting Struct Using Comparator Function
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+#define FastIO ios_base::sync_with_stdio(0); cin.tie(0)
```

Comment:
up to you but I would rather have this line inline in the setIO method

Reply:
I prefer it separate.